spec/integration/local_spec.rb

```diff
@@ -33,6 +33,12 @@
       expect(result[0]['_error']).to be
     end
 
+    it 'returns correct exit code' do
+      result = run_failed_nodes(%W[command run exit(2) -t #{uri}]).first
```

Comment:
This case already does the right thing without this change. `exit` is a keyword in powershell so it does the right thing (causes powershell.exe to exit 2), but the failure happens when you invoke another process in powershell and that process exits 2.

```
$ bolt command run 'exit(2)' -t localhost
The command failed with exit code 2
$ bolt command run 'powershell.exe -Command exit(2)' -t localhost
The command failed with exit code 2
$ bolt command run 'powershell.exe -Command "powershell.exe -Command exit(2)"' -t localhost
The command failed with exit code 1
```

The first and second cases are actually identical, since we automatically wrap the bare `exit(2)` in a call to `powershell.exe -Command exit(2)`. But if you wrap _that_ in another powershell invocation, the outer invocation exits 1 because it throws away the exit code from the inner invocation.